force-app/main/default/lwc/customEventButton/customEventButton.js

```diff
@@ -0,0 +1,96 @@
+import { LightningElement, track } from "lwc";
+import {
+  APPLICATION_SCOPE,
+  createMessageContext,
+  releaseMessageContext,
+  subscribe as subscribeMessageChannel,
+  unsubscribe as unsubscribeMessageChannel
+} from "lightning/messageService";
+import MC_FIRE_CUSTOM_EVENT from "@salesforce/messageChannel/msmxSheet__fireCustomEvent__c";
+import MC_SELECT_RECORDS from "@salesforce/messageChannel/msmxSheet__selectRecords__c";
+/**
+ *
+ */
+const MESSAGE_CHANNELS = {
+  fireCustomEvent: MC_FIRE_CUSTOM_EVENT,
+  selectRecords: MC_SELECT_RECORDS
+};
+/**
+ *
+ */
+
+export default class CustomEventButton extends LightningElement {
+  messageContext;
+
+  subscriptions = [];
+
+  @track totalPrice = 0;
```

Comment:
`@track` decorator is not needed in latest LWC.
https://help.salesforce.com/s/articleView?id=release-notes.rn_lwc_track.htm&release=224&type=5